cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaSetOperator.java

```diff
@@ -72,4 +89,55 @@ public static boolean needsRollingUpdate(StatefulSetDiff diff) {
                 .rollingRestart(podNeedsRestart);
     }
 
+    /**
+     * @param sts Stateful set to which kafka pod belongs
+     * @param pod Specific kafka pod
+     * @return a future which contains map with all kafka properties including their values
+     */
+    public Future<Map<ConfigResource, Config>> getCurrentConfig(StatefulSet sts, Pod pod) {
```

Comment:
Can we keep this somewhere else? I think it would be great to keep the Kubernetes functions and the Kafka functions separate.